src/Analyzers/Analyzers.csproj

```diff
@@ -29,6 +29,11 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="3.3.4" PrivateAssets="all" />
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="4.9.2" PrivateAssets="all" />
+    <PackageReference Include="Microsoft.CodeAnalysis.ResxSourceGenerator" Version="3.11.0-beta1.24165.2" PrivateAssets="all" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <EmbeddedResource Update="*.resx" />
```

Comment:
This can either be removed.

Unless you need the format methods, in which case add `EmitFormatMethods="true"`. Also change to `**/*.resx` so it will apply to resx files recursively.

``` csharp
    <EmbeddedResource Update="**/*.resx" EmitFormatMethods="true" />
```

But seeing how you pass to `LocalizableString` type, I don't think you need the format methods.

Reply:
nice, removed :)